src/routes/Main.js

```diff
@@ -240,10 +240,18 @@ class Main extends Component {
       },
     } = this.props;
     const users = get(resources, 'users.records', []);
+    const vendors = get(resources, 'vendors.records', []);
     const resultsFormatter = {
       'poNumber': order => get(order, 'poNumber', ''),
+      'vendorCode': order => {
+        const vendorId = get(order, 'vendor', '');
+        const vendorCode = get(vendors.find(({ id }) => id === vendorId), 'code', '');
+
+        return vendorCode.code;
```

Comment:
`return get(vendors.find(({ id }) => id === vendorId), 'code', '')`?

Reply:
Yes, thank you. @aliaksei-chumakou suggested the same on Friday